python/tests/test_chain.py

```diff
@@ -120,7 +121,8 @@ def test_panda_FK(self):
             [0, -1, 0],
             [0, 0, -1]
         ])
-        expected_sM7 = Pose3(sR7, Point3(0.0882972, 0.00213401, 0.933844))
+        #regression
+        expected_sM7 = Pose3(sR7, Point3(0.098517, 0.004252, 0.971403))
```

Comment:
can you clarify what's going on with the regressions in this file?  i.e. was it wrong before or did somehow the fixed joint change stuff?  or is it possible this is from a different PR and this is just a weird diff artifact?

Reply:
Links connected by fixed joints were getting merged into one link, which meant that the CoM was being changed to the weighted mean of the two. This is no longer the case.